src/main/java/io/cloudsoft/terraform/entity/ManagedResourceImpl.java

```diff
@@ -46,6 +46,7 @@ public void updateResourceState() {
                     "TF-ASYNC", "Resource changed outside terraform.");
         } else {
             sensors().set(Attributes.SERVICE_STATE_ACTUAL, Lifecycle.CREATED);
+            ServiceStateLogic.updateMapSensorEntry(this, Attributes.SERVICE_PROBLEMS, null, null);
```

Comment:
service.problems does not automatically clear when service.running is set to "CREATED"

Reply:
good catch; we should provide the key but i'll tweak this